Library/Formula/svmlight.rb

```diff
@@ -0,0 +1,16 @@
+class Svmlight < Formula
+  homepage "http://svmlight.joachims.org/"
+  url "http://download.joachims.org/svm_light/current/svm_light.tar.gz"
+  version "6.02"
+  sha1 "3b0f843e6f8f4f168d3e33ab957f22c56ccadd86"
+
+  def install
+    system "make all"
+    bin.install "svm_learn"
+    bin.install "svm_classify"
+  end
+
+  test do
+    system "false"
```

Comment:
No tests are built in.  Should I add a file that can be read for testing?  Where should this file live?  I'm not sure what is normally done.  Could you point me to a formula that installs a file for reading during testing that is independent of the software's official release?